Assets/Scripts/Game/InputManager.cs

```diff
@@ -1199,6 +1199,28 @@ public KeyCode ParseKeyCodeString(String s)
             }
         }
 
+        /// <summary>
+        /// Calculates UI scroll movement value based on analog axes input.
+        /// </summary>
+        public float GetUIScrollMovement()
+        {
+            float horizontal = Input.GetAxis(cameraAxisBindingCache[0]);
+            float vertical = Input.GetAxis(cameraAxisBindingCache[1]);
+
+            if (GetAxisActionInversion(AxisActions.CameraHorizontal))
+                horizontal *= -1;
+
+            if (GetAxisActionInversion(AxisActions.CameraVertical))
+                vertical *= -1;
+
+            float calculated_deadzone = Mathf.Sqrt(horizontal * horizontal + vertical * vertical);
+            if (calculated_deadzone <= JoystickDeadzone)
+                return 0.0f;
+
+            // Use both analog axes for scrolling movement - for now it doesn't matter which axis we use for scrolling
+            return horizontal + vertical;
```

Comment:
Not sure if this was your preference, but with my controller (Xbox on Windows/Linux, aimed for as the standard), I get inverse movement where Up moves it down and Down moves it up, but I think it's more standard for joysticks that they move in the same direction. Can you do `return -(horizontal + vertical);`?

Reply:
You're right, I was so excited to get it working, that I didn't care which direction it scrolled.

I inverted only the vertical axis - I tested it now on Deck (I assume it should mimic Xbox controller, as it's configured to do so in the options) and now it scrolls in the following directions:
```
stick up - scrolls up
stick down - scrolls down

stick left - scrolls up
stick right - scroll down
```

Do we want to invert horizontal axis too? Current behavior looks most intuitive to me.